tests/unit/manage/test_views.py

```diff
@@ -1292,16 +1292,7 @@ def test_post_new_role(self, monkeypatch, db_request):
             pretend.call("Added collaborator 'testuser'", queue="success"),
         ]
 
-        assert send_collaborator_added_email.calls == [
-            pretend.call(
-                db_request,
-                user,
-                db_request.user,
-                project.name,
-                form_obj.role_name.data,
-                []
-            )
-        ]
+        assert send_collaborator_added_email.calls == []
```

Comment:
We probably still want this assertion to stay or a similar assertion to replace it.